samples/boards/nrf/nrf_led_matrix/boards/bbc_microbit.overlay

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2021, Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	led_matrix {
```

Comment:
can this be added to the board, as for v2?

Reply:
Done.

Reply:
This overlay is no longer needed then.

Reply:
Yes, but I kept it is a template, as per:
https://github.com/zephyrproject-rtos/zephyr/blob/c3dd5dc6071b6e82d5dbd6395b38e6bfb25911f5/samples/boards/nrf/nrf_led_matrix/README.rst#L20-L21
That was the reason I haven't originally added the `led_matrix` node to the bbc_microbit board definition. Should I remove the overlay then and that sentence from README?

Reply:
why not document it on the readme then? having an overlay is a bit confusing if not required.

Reply:
I think the best place for that is the binding as an example (main description field)

Reply:
> why not document it on the readme then? having an overlay is a bit confusing if not required.

I agree, it's confusing. I'll remove the overlay.